library/modules/Textures.cpp

```diff
@@ -0,0 +1,208 @@
+#include "Internal.h"
+
+#include "modules/Textures.h"
+
+#include "Debug.h"
+#include "PluginManager.h"
+
+#include "df/enabler.h"
+
+using df::global::enabler;
+using namespace DFHack;
+
+namespace DFHack {
+    DBG_DECLARE(core, textures, DebugCategory::LINFO);
+}
+
+static bool g_loaded = false;
+static long g_num_dfhack_textures = 0;
+static long g_dfhack_logo_texpos_start = -1;
+
+static DFLibrary *g_sdl_handle = nullptr;
+static DFLibrary *g_sdl_image_handle = nullptr;
+static const std::vector<std::string> SDL_LIBS {
+    "SDLreal.dll",
+    "SDL.framework/Versions/A/SDL",
+    "SDL.framework/SDL",
+    "libSDL-1.2.so.0"
+};
+static const std::vector<std::string> SDL_IMAGE_LIBS {
+    "SDL_image.dll",
+    "SDL.framework/Versions/A/SDL_image",
+    "SDL.framework/SDL_image",
```

Comment:
```suggestion
    "SDL_image.framework/Versions/A/SDL_image",
    "SDL_image.framework/SDL_image",
```
confirmed by downloading `df_47_05_osx.tar.bz2`